### DISCUSSION TITLE
Passing a variable from the client sdk to function

### DISCUSSION THREAD

**Discussion by issasaleh** [appwrite/appwrite#5869] [Q&A]
Hi,
I am a beginner in programming,

Can I pass a variable from the client SDK to a function (node js) and then execute the function? And how can I do that?

thanks in advance,

**Comment by wiledal** [comment]
Indeed you can!

As outlined here, https://appwrite.io/docs/client/functions?sdk=web-default#functionsCreateExecution. 
You call 
```js
functions.createExecution('YOUR FUNCTION ID', 'YOUR DATA')
```

The data will be available in the function:
```js
module.exports = async  function (req, res) {
  const data = req.variables.APPWRITE_FUNCTION_DATA; // "YOUR DATA"
}
```
